src/OrchardCore.Modules/Orchard.Admin/Orchard.Admin.csproj

```diff
@@ -18,4 +18,8 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="1.0.0" />
```

Comment:
This to be able to run dotnet watch in each module directory we want to watch.

I tried it by only referencing the tool in e.g `Orchard.Cms.Web`. Then going to a module directory, then calling the tool by specifying  the `Orchard.Cms.Web` path. But, as i remember i had some problems because of the working directory of the tool.

I will re-try it.